mlflow/store/tracking/file_store.py

```diff
@@ -1319,3 +1326,372 @@ def _read_helper(root, file_name, attempts_remaining=2):
                 return _read_helper(root, file_name, attempts_remaining - 1)
 
         return _read_helper(root, file_name, attempts_remaining=retries)
+
+    def start_trace(
+        self,
+        experiment_id: str,
+        timestamp_ms: int,
+        request_metadata: Dict[str, str],
+        tags: Dict[str, str],
+    ) -> TraceInfo:
+        """
+        Start an initial TraceInfo object in the backend store.
+
+        Args:
+            experiment_id: String id of the experiment for this run.
+            timestamp_ms: Start time of the trace, in milliseconds since the UNIX epoch.
+            request_metadata: Metadata of the trace.
+            tags: Tags of the trace.
+
+        Returns:
+            The created TraceInfo object.
+        """
+        request_id = request_metadata.get(SpanAttributeKey.REQUEST_ID)
+        if request_id is None:
+            raise MlflowException("Request ID is required to start a trace.", INTERNAL_ERROR)
```

Comment:
I think file store should generate request ID rather than needing upstream to generate it, otherwise upstream needs to handle difference when calling `start_trace()` (rest store generates request ID while this won't). We've used `uuid.uuid4().hex` for SQLAlchemyStore and eval integration, maybe we can make it a shared function.